### ISSUE TITLE
Get a reference to marker markup.

### ISSUE THREAD

**Issue by gurmukhp** [marmat/google-maps-api-addons#29]
Hi,

I'd like to be able to get the marker HTML so I'm able to add an event listener and also get the position of it. Unfortunately, this.marker_ is created from the onAdd method which isn't evaluated immediately after the constructor is called. Would it be possible to add a callback from onAdd? Maybe a callback could be defined in the options?

If you agree with this approach, I'd be happy to arrange a pull request.

Thanks!

**Comment by marmat** [comment]
I think you should be able to use the Promise ([compatibility](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise#Browser_compatibility)) returned by `setPano()`: [panomarker.js#L598](https://github.com/marmat/google-maps-api-addons/blob/master/panomarker/src/panomarker.js#L598). In that case you'll want to call `setPano` explicitly rather than passing in a pano in the constructor options.

**Comment by gurmukhp** [comment]
That would mean setPano would be called twice, once from the constructor and again by calling setPano explicitly. 

IE doesn't support Promise natively, I could use goog.promise but PanoMarker only checks for native support. Could the markup for the marker be retrieved in IE even if Promises are unsupported?

**Comment by marmat** [comment]
Yes, though the first call would be `setPano(null)` which is essentially a no-op.

We could [`google.maps.event.trigger(...)`](https://developers.google.com/maps/documentation/javascript/3.exp/reference#event) an event once `PanoMarker.onAdd()` finishes (and we have created the marker DOM node), maybe even with a reference to the marker div as a parameter to the listener. This should be fairly browser-agnostic. Would that work for your use case? 

You would probably still want to call `setPano` explicitly in order to avoid races with the event triggering before you're able to attach the listener, i.e.:

```
var marker = new PanoMarker(...);
google.maps.event.addListener(marker, 'add', myOnAddHandlerFn);
marker.setPano(...);
```

**Comment by gurmukhp** [comment]
That works perfectly. I'll raise a PR for this, thank you :)

**Comment by marmat** [comment]
Fixed in #28.
